code-challenges.js

```diff
@@ -8,62 +8,160 @@
 
 // Pseudo code:
+//-input is 2 arrays 
+//-output return length of both arrays combined using .length built in method
+//-process create a function combinedArrays and set parameters to combine arrays
+//-return the output of the combined arrays a + b
+const combinedArrays = (a,b) => {
+    return a + b
+}
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+console.log(combinedArrays(padres1984WorldSeriesRuns.length, padres1998WorldSeriesRuns.length))
+//output: 9
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
+//-input is the test variable Charlie 2023
+//-output is the string reversed
+//-process create a function called stringReversed and set only 1 parameter to equal the number of test variables, then turn string into an array, reverse the array and join the array back into a string
+//-return the string reversed using the .split built in method to create the array, .reverse to reverse the array and then use the .join built in method to join the array into a string
+
+const stringReversed = (a) => {
+    return a.split("").reverse().join("")
+
+}
 
 const currentCohort = "Charlie 2023"
 // Expected output: "3202 eilrahC"
 
+
+console.log(stringReversed(currentCohort))
+  //output" 3202 eilrahC
+
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
+// -input is the array numberOfConnections
+// -output is the last index of the givenValue1 and givenValue2
+// -process is to create a function named lastInstance, search numberOfConnections for the last index of givenValue1 and givenValue2 using the .lastIndexOf built in method give it a single parameter and use the function to index the given value
+// -return the expected output of givenValue1 and givenValue2 respectively
+
+const lastInstance = (a) => {
+    return numberOfConnections.lastIndexOf(a)
+}
 
 const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 
 const givenValue1 = 42
 // Expected output: 7
 
+console.log(lastInstance(givenValue1))
+  //output: 7
+
 const givenValue2 = 10
 // Expected output: 8
+console.log(lastInstance(givenValue2))
+  //output: 8 
 
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
+// -input is 2 arrays, sanDiegoSummerTemperatures and sanDiegoWinterTemperatures 
+// -output is the 2 arrays sorted from largest to smallest
+// -process is too create a function named tempOrder with a parameter that equals the test variables and using the .sort and .reverse manipulators sort the test variables from largest to smallest
+//-return sanDiegoSummerTemperatures and sanDiegoWinterTemperatures sorted in descending order
+
```

Comment:
The input is not 2 arrays, but 1. Make sure you explain what the `.sort` and `.reverse` methods are doing in your function to achieve the output instead of just stating that they sort -- how do they produce the output?